murageden/converge-level-up
# #155820624 feature(launch detail activity): launch detail activity from the main…

## What does this PR do?
- App Basics
- Launch the DetailActivity from the MainActivity

## Description of Task to be completed?
- Launch the DetailActivity from the MainActivity

## How should this be manually tested?
- Clone the repository and set up the Android Studio as described on the `README`
- Run the application
- The app should run as shown on the video below

## What are the relevant pivotal tracker stories?
155820624

## Screenshots (if appropriate)?
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/22506247/43589118-b79c3a8e-9676-11e8-8a3d-73b5cc6eb676.gif)



## Questions:
N/A
